chore: changed some of the wording in the overview page (#106)
c6o/docs: content/welcome/overview.md

# Overview

> CodeZero is a Modern Development Platform for Kubernetes. Before Kubernetes and Cloud computing, for most software development it has been possible to write and debug software on a single developer workstation. As software complexity grows and organizations move to Microservices Architectures designed to run on clusters of computers, it becomes no longer feasible to run these applications on a single workstation. This makes developing new features and diagnosing issues challenging.

Reviewer: ```suggestion
CodeZero is a modern development platform for Kubernetes. Before Kubernetes and Cloud-computing it was possible to write and debug _most_ software on a single developer workstation. As software complexity has grown and organizations have moved to a microservices architectures designed to run on clusters of computers, it is no longer feasible to run these applications on a single workstation. This makes developing new features and diagnosing issues challenging.
```
